charts/platform-secrets/templates/rbac.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ include "platformSecrets.fullname" . }}-sa
```

Comment:
I would remove suffixes such as sa, cr, crb as the resource kind already includes this information